pkg/skaffold/kubernetes/port_forward.go

```diff
@@ -74,13 +75,21 @@ var (
 
 // Forward port-forwards a pod using kubectl port-forward
 // It returns an error only if the process fails or was terminated by a signal other than SIGTERM
-func (*kubectlForwarder) Forward(parentCtx context.Context, pfe *portForwardEntry) error {
+func (*kubectlForwarder) Forward(parentCtx context.Context, pfe *portForwardEntry, address string) error {
```

Comment:
Shouldn't you be taking the address from the portForwardEntry?

Reply:
Yeah but its a global setting not per pod, so it was easier to add it here. Maybe its better that this is actually part of the skaffold.yaml file but thats way more work :p

Reply:
I think previously skaffold had the settings for port-forwarding in the skaffold.yaml file, but it was removed (because of the default case should work fine). But in our case we need --address... So i guess commandline is good enough...